src/components/controls/form-controls/DatePicker.tsx

```diff
@@ -1,24 +1,26 @@
-import React, { Component } from 'react';
+import React from 'react';
 import moment from 'moment';
-import { t } from '../../utils';
-import { EnhanceInputWithLabel } from '../../enhancers/EnhanceInputWithLabel';
-import { default as ReactDatePicker } from 'react-datepicker';
-import { BaseInputProps } from './inputs/BaseInput';
+import {default as ReactDatePicker} from 'react-datepicker';
+import {t} from '../../utils';
+import {EnhanceInputWithLabel} from '../../enhancers/EnhanceInputWithLabel';
+import {BaseInputProps} from './inputs/BaseInput';
 
-type DatePickerProps = BaseInputProps<moment.Moment | null>;
+type DatePickerProps = BaseInputProps<moment.Moment | null> & {
+  showMonthYearPicker?: boolean;
+  dateFormat?: string;
+};
 
-export const DatePicker = EnhanceInputWithLabel(class extends Component<DatePickerProps> {
-  render() {
-    return (
-      <ReactDatePicker
-        className="form-control"
-        selected={this.props.value ? this.props.value.toDate() : undefined}
-        onChange={dateString => this.props.onChange(dateString ? moment(dateString) : null)}
-        locale="nl"
-        dateFormat="dd/MM/yyyy"
-        peekNextMonth={false}
-        todayButton={t('controls.today')}
-      />
-    );
-  }
+export const DatePicker = EnhanceInputWithLabel((props: DatePickerProps) => {
+  return (
+    <ReactDatePicker
+      className="form-control"
+      selected={props.value ? props.value.toDate() : undefined}
+      onChange={dateString => props.onChange(dateString ? moment(dateString) : null)}
+      locale="nl"
+      dateFormat={props.dateFormat || 'dd/MM/yyyy'}
+      peekNextMonth={false}
+      todayButton={t('controls.today')}
+      showMonthYearPicker={props.showMonthYearPicker}
+    />
+  );
```

Comment:
En hier zitten we dus met merge conflicten :)

Ik heb hier de "nl" en de "dd/MM/yyyy" in constanten gestopt.
Dus de PR die het laatst gemerged wordt zal het conflict moeten oplossen :)